packages/salesforcedx-vscode-core/src/commands/forceStartApexDebugLogging.ts

```diff
@@ -115,7 +116,19 @@ export class ForceStartApexDebugLoggingExecutor extends SfdxCommandletExecutor<{
 
 export async function getUserId(projectPath: string): Promise<string> {
   const defaultUsernameOrAlias = await getDefaultUsernameOrAlias();
-  const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias!);
+  if (isNullOrUndefined(defaultUsernameOrAlias)) {
+    const err = nls.localize('error_no_default_username');
+    telemetryService.sendError(err);
+    throw new Error(err);
+  }
+
+  const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
+  if (isNullOrUndefined(username)) {
+    const err = nls.localize('error_no_default_username');
```

Comment:
The error we throw here is the same as in line 120. We won't be able to distinguish from the telemetry data which condition failed.

Reply:
That's true, I can make them unique for telemetry purposes but keep it as is for the end user since the message still makes sense in both situations.